WSSiOS/WSSiOS/App/SceneDelegate.swift

```diff
@@ -45,6 +57,15 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
                           completion: nil)
     }
     
+    func checkIsRegistered() {
+        if !APIConstants.isRegister {
+            UserDefaults.standard.setValue(nil,
+                                           forKey: StringLiterals.UserDefault.accessToken)
+            UserDefaults.standard.setValue(nil,
+                                           forKey: StringLiterals.UserDefault.refreshToken)
```

Comment:
이렇게도 쓸 수 있어요! 동작은 비슷해서 코드 스탈 따라가면 될 거 같아요
```suggestion
            UserDefaults.standard.removeObject(forKey: StringLiterals.UserDefault.accessToken)
            UserDefaults.standard.removeObject(forKey: StringLiterals.UserDefault.refreshToken)
```

Reply:
고마워요 !! 지금 저렇게 적용해둔 코드가 너무 많아서 다음에 시도해볼게요! 아에 삭제하는게 훨씬 의미상 맞겠네요 !!